kotlin/internal/jvm/compile.bzl

```diff
@@ -395,6 +396,7 @@ def _run_kt_builder_action(
     args.add_all("--deps_artifacts", deps_artifacts, omit_if_empty = True)
     args.add_all("--kotlin_friend_paths", associates.jars, map_each = _associate_utils.flatten_jars)
     args.add("--instrument_coverage", ctx.coverage_instrumented())
+    args.add("--kapt_correct_error_types", toolchains.kt.kaptInfo[_KaptInfo].correct_error_types)
```

Comment:
We should just pass all the options in here like we do for Kotlinc/Javac/Ksp, otherwise we'd end up having to create new proto entries for every single kapt option that we want to support.